composer.json

```diff
@@ -40,8 +40,8 @@
         "sonata-project/media-bundle": "For using the admin media browser."
     },
     "conflict": {
-        "sonata-project/admin-bundle": "<3.0 || >= 4.0",
-        "sonata-project/media-bundle": "<3.0 || >= 4.0"
+        "sonata-project/admin-bundle": "<3.0 || >=5.0",
```

Comment:
For me it is better because it will avoid troubles like the one we are trying to solve here and if there is a need in the future for the restriction we can add it then. WDYT @greg0ire ?